src/Symfony/Component/Validator/Constraints/IsNull.php

```diff
@@ -35,6 +37,12 @@ class IsNull extends Constraint
 
     public $message = 'This value should be null.';
 
+    /**
+     * @param array<string,mixed>|null $options The options as an associative array
```

Comment:
Why aren't the arguments aligned here when they were before?

Reply:
Hmm you mean when each part of the param doc starts at the same column?